src/main/java/com/searchservice/app/domain/service/InputDocumentService.java

```diff
@@ -64,7 +64,7 @@ private UploadDocumentUtil documentUploadResponse(String tableName, String paylo
 	public ThrottlerResponse addDocuments(boolean isNRT,String tableName, String payload) {
 
 		if (!manageTableServicePort.isTableExists(tableName))
-			throw new BadRequestOccurredException(400, tableName.split("_")[0] + " table doesn't exist");
+			throw new CustomExceptionHandler(400,HttpStatusCode.BAD_REQUEST_EXCEPTION,tableName.split("_")[0] + " table doesn't exist");
```

Comment:
Please use the custom ErrorStatus from HttpStatusCode for table "does not exist"